mne/io/edf/edf.py

```diff
@@ -79,10 +124,9 @@ class RawEDF(BaseRaw):
     been made to decode these in MNE.
 
     For GDF files, the stimulus channel is constructed from the events in the
-    header. You should use keyword ``stim_channel=-1`` to add it at the end of
-    the channel list. The id numbers of overlapping events are simply combined
-    through addition. To get the original events from the header, use method
-    ``raw.get_gdf_events``.
+    header. The id numbers of overlapping events are simply combined through
+    addition. To get the original events from the header, use method
+    ``raw.get_edf_events``.
```

Comment:
this should change to the method (if you change it so classes are documented) or the function (if you keep it as-is)